src/pages/CompanyTrainingOverview/components/CourseListItem.jsx

```diff
@@ -0,0 +1,64 @@
+import { useTheme } from '@emotion/react';
+import { Grid, ListItem, ListItemText } from '@mui/material';
+import PropTypes from 'prop-types';
+
+function CourseListItem({ courseName, courseCategory, trainingNumber }) {
+  const theme = useTheme();
+
+  return (
+    <Grid item xs={6}>
+      <ListItem
+        style={{
+          height: '150px',
+          background: theme.palette.primary.light,
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          justifyContent: 'flex-start',
+        }}
+      >
+        <ListItemText
+          primary={courseCategory}
+          primaryTypographyProps={{
+            variant: 'body1',
+            sx: {
```

Comment:
Lower resolutions cause the text to overflow here. You could add something like this to stop that from happening 
```suggestion
            sx: {
            wordWrap: "break-word",
            width: "70%",
```
. You could also use a grid.